test/exe/Main.hs

```diff
@@ -1714,16 +1733,24 @@ addFunctionConstraintTests = let
   in testGroup "add function constraint"
   [ check
     "Add `Eq a` to the context of the type signature for `eq`"
-    (missingConstraintSourceCode Nothing)
```

Comment:
I think that not using this Maybe indirection makes it more immediately obvious what's the difference between "before" and "after" (+ bonus: it makes the test source code few lines shorter :smile: )